changelog/6000.bugfix.rst

```diff
@@ -0,0 +1 @@
+this is a fake entry with a bad :ref:`reference-that-should-fail`
```

Comment:
don't forget to remove that :-D

Reply:
Haha, I can't merge without fixing it anyway, that's the point! 😄